Sources/BraintreeCore/BTShopperInsightsRequest.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+/// Buyer data required to use the Shopper Insights feature.
+/// - Note: This feature is in beta. It's public API may change or be removed in future releases.
+@objcMembers public class BTShopperInsightsRequest: NSObject {
+    
+    /// The buyer's email address.
+    public let email: String
+    
+    /// The buyer's country code prefix to the national telephone number. An identifier for a specific country.
+    /// Must not contain special characters.
+    public let phoneCountryCode: String
+    
+    /// The buyer's national phone number. Must not contain special characters.
+    public let phoneNationalNumber: String
+    
+    /// Initialize a `BTShopperInsightsRequest`
+    /// - Parameters:
+    ///   - email: The buyer's email address.
+    ///   - phoneCountryCode: The buyer's country code prefix to the national telephone number. An identifier for a specific country.
+    ///   - phoneNationalNumber: The buyer's national phone number.
```

Comment:
nit to add the same docstring here:
```suggestion
    ///   - phoneCountryCode: The buyer's country code prefix to the national telephone number. An identifier for a specific country. Must not contain special characters.
    ///   - phoneNationalNumber: The buyer's national phone number. Must not contain special characters.
```